tests/seed-array.test.ts

```diff
@@ -1,7 +1,7 @@
-import { JSONSchema7 } from 'json-schema'
 import { seedArray } from '../src/seed-array.js'
+import type { JSONSchema } from '../src/types.js'
 
-it.each<[string, JSONSchema7, unknown[]]>([
+it.each<[string, JSONSchema, unknown[]]>([
```

Comment:
Is this type change necessary because the previous schema type doesn't include the `example` property?

Reply:
Yep, OpenAPI spec extends the JSONSchema drafts